dts/arm/st/h5/stm32h563.dtsi

```diff
@@ -27,12 +27,13 @@
 			clocks = <&rcc STM32_CLOCK(AHB1, 19)>;
 
 			mac: ethernet {
-				compatible = "st,stm32h7-ethernet", "st,stm32-ethernet";
+				compatible = "st,stm32h5-ethernet", "st,stm32-ethernet";
 				interrupts = <106 0>;
 				clock-names = "mac-clk-tx", "mac-clk-rx";
 				clocks = <&rcc STM32_CLOCK(AHB1, 20)>,
 					 <&rcc STM32_CLOCK(AHB1, 21)>;
 				status = "disabled";
+				memory-regions = <&sram2>;
```

Comment:
Could you remove this property. H5 is currently expected to locate ethernet buffer in Zephyr main RAM.